Hunted Mobile/Hunted Mobile/Hunted Mobile/ViewModel/MapViewModel.cs

```diff
@@ -636,8 +636,12 @@ public class MapViewModel : BaseViewModel {
             mapViewService.AddPlayerPin();
             mapViewService.AddPoliceStationPin(gameModel.PoliceStationLocation);
 
-            foreach(var user in mapModel.Players) {
-                mapViewService.AddTeamMatePin(user);
+            foreach(var thief in mapModel.Thiefs) {
+                mapViewService.AddTeamMatePin(thief);
+            }
+
+            foreach(var police in mapModel.Police) {
+                mapViewService.AddTeamMatePin(police);
             }
```

Comment:
Volgens mij is dit niet nodig

Reply:
Ik wil graag de lijst met `Thiefs` ophalen, want die wordt nu gefilterd op `IsCaught`. De andere optie is dan om een if-statment op te stellen die controleert voor politie en dieven, dan ziet er dan als volgt uit: `!user.IsCaught && user is Thief || user is Police`. Mijn voorkeur gaat dan toch naar de gefilterde lijst

Reply:
Oja dat is waar. Dan kun je het zo laten.